.github/container/manifest.yaml

```
@@ -148,3 +148,8 @@ language-to-reward-2023:
   tracking_ref: main
   latest_verified_commit: abb8e5125e4ecd0da378490b73448c05a694def5
   mode: git-clone
+mlperf-logging:
+  url: https://github.com/mlcommons/logging.git
+  tracking_ref: main 
+  latest_verified_commit: 38709131757a786d7f66150243b49eb0365324d7
+  mode: git-clone
```

You can set it to `pip-vcs` to document that this isn't source that we clone, but used just during pip-compile + pip-sync
```suggestion
  mode: pip-vcs
```

---

changed it to pip-vcs